blog/tests/models_test.py

```diff
@@ -0,0 +1,64 @@
+"""Tests for Post model's Blog app."""
+
+from django.contrib.auth.models import User
+from django.test import TestCase
+
+from blog.models import Post
+
+
+class PostModelTest(TestCase):
+    """Tests for Post model."""
+
+    @classmethod
+    def setUpTestData(cls):
+        """Set up non-modifiable objects used by all test methods."""
+        Post.objects.create(author=User.objects.create(username='bob'),
+                            title='Title', text='A few lines of text')
+
+    def test_author_label(self):
+        """Test for author label."""
+        post = Post.objects.get(id=1)
+        field_label = post._meta.get_field('author').verbose_name
```

Comment:
What is the point of this test? What are you testing?

Reply:
Removed too